src/datasources/CartsDataSource.ts

```diff
@@ -16,15 +16,14 @@ export class CartsDataSource extends RESTDataSource {
   }
 
   async getCart(id) {
-    const { data } = await this.get(`/carts/${id}/`);
+    const results = await this.get(`/carts/${id}`, {
+      include: 'items'
```

Comment:
This assumes that people will always ask for items when fetching a cart. Which I guess is an okay assumption to make.